# The Hand of Chandra

![](data/234_q0.jpg)

A small, dense object only twelve miles in diameter is responsible for this beautiful X-ray nebula that spans 150 light years and resembles a human hand! At the center of this image made by NASA's [Chandra X-ray Observatory](https://en.wikipedia.org/wiki/Chandra_X-ray_Observatory) is a very young and powerful pulsar known as [PSR B1509-58](https://en.wikipedia.org/wiki/PSR_B1509-58).

The **pulsar** is a rapidly spinning neutron star which is spewing energy out into the space around it to create complex and intriguing structures, including this one that resembles a large, cosmic hand.

Astronomers think that the pulsar and its nebula are about 1,700 years old, and are located about **17.000 light years away** (e.g., **5,000 parsecs**). Finger-like structures extend to the north, apparently energizing knots of material in a neighboring gas cloud known as [RCW-89](http://galaxymap.org/cat/view/rcw/89). The transfer of energy from the wind to these knots makes them glow brightly in X-rays (orange and red features to the upper right).

-----------------------------------------

### Question 1

The _field of view_ is **19 arcminutes** across. Using similar triangles and proportions, if 1 arcminute at a distance of 1,000 parsecs equals a length of 0.3 parsecs, how wide is the image in parsecs?

### Answer 1

If **0.3 parsecs** seen from a distance of **1,000 parsecs** subtends an angle of **1 arcminute**, then **1 arcminute** at a distance of **5,200 parsecs** will subtend:

$ \frac{x}{0.3} = \frac{5,200}{1,000} $

In [2]:
# 1 arcminute, in parsecs.
q1_parsecs = (5200 / 1000) * 0.3
q1_parsecs

1.56

If the value of **1 arcminute** is equal to **1.56 parsecs**, then **19 arcminutes** will subtend:

$ \frac{x}{1.56} = \frac{19}{1} $

In [6]:
# Width of the image, in parsecs.
q1_width = (19 / 1) * 1.56
q1_width

29.64

------------------------------------------

### Question 2

Measure the width of this image with a millimeter ruler. What is the scale of this image in parsecs per millimeter? How far, in light years, is the bright spot in the 'palm' where the pulsar is located from the center of the ring-like knots in RCW 89?

(_1 parsec = 3.26 light years. Round your answer to the nearest light year._)

### Answer 2

The width of the image is about **96 millimeters**. The scale is then:

In [9]:
# Scale of the image, in parsecs per millimeter.
q2_scale = 29.64 / 96

The distance from the bright spot to the ring of knots is about **40 millimeters**. At the scale of the image, this equals:

In [10]:
# Parsecs from the bright spot to the ring of knots.
q2_parsecs = 40 * q2_scale

# Conversion to light years.
q2_parsecs * 3.26 

40.261

------------------------------------------

### Question 3

If the speed of the plasma is **10,000 km/sec**, how many years did it take for the plasma to reach RCW-89 if 1 light year = $9.5 x 10^{12}$ kilometers, and there are $3.1 x 10^7$ seconds in 1 year?

### Answer 3

$ time = \frac{distance}{speed} $

First, convert light years to kilometers:

In [16]:
q3_distance = 40.261 * (9.5 * 10**12)
q3_distance

382479500000000.0

Then, divide the distance by the speed to get the time, and convert to years.

In [19]:
q3_time = q3_distance / 10000
q3_time / (3.1 * 10**7)

1233.8048387096774

Based on our back-of-the-envelope calculations, it took the plasma around 1,234 years to reach RCW-89. Wow!